.github/workflows/wpcloud.yml

```
@@ -13,6 +13,7 @@ jobs:
   build:
     name: Install the Monorepo and build wpcomsh
     runs-on: ubuntu-latest
+    if: github.repository == 'Automattic/jetpack'
```

This works, but to be a little friendlier to forks that want to actually set up secrets we usually do something like this
```suggestion
    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
```